packages/manager/modules/pci/src/projects/project/storages/databases/database/connectors/add-connector/add-connector.controller.js

```diff
@@ -37,6 +38,12 @@ export default class AddConnectorCtrl {
     this.messages = this.messageHandler.getMessages();
   }
 
+  areRequiredFieldsFilled() {
+    return this.requiredFields.every(
+      ({ name }) => this.model[name] != null && this.model[name] !== '',
```

Comment:
**suggestion**: isn't this enough?
```suggestion
      ({ name }) => this.model[name],
```

Reply:
Same as here: https://github.com/ovh/manager/pull/6725/files/17460da7e485d71c364eb97dfd20d2dbdaa726a7#r834291469